pyrene/src/components/CheckboxPopover/CheckboxPopover.jsx

```diff
@@ -33,13 +33,12 @@ export default class CheckboxPopover extends React.Component {
           onClickOutside={() => this.setState({ displayPopover: false })}
           renderPopoverContent={() => <CheckboxList listItems={this.props.listItems} onItemClick={this.props.onItemClick} onRestoreDefault={this.props.onRestoreDefault} />}
         >
-          <div styleName={clsx('popoverTriggerButton', { popoverOpen: this.state.displayPopover })} onClick={this.togglePopover}>
-            <div styleName="buttonLabel" className="unSelectable">
+          <div className={clsx(styles.popoverTriggerButton, { [styles.popoverOpen]: this.state.displayPopover })} onClick={this.togglePopover}>
+            <div className={clsx(styles.buttonLabel, 'unSelectable')}>
               {this.props.buttonLabel}
             </div>
             <div
-              styleName="arrowIcon"
-              className={this.state.displayPopover ? 'pyreneIcon-chevronUp' : 'pyreneIcon-chevronDown'}
+              className={clsx('arrowIcon', { 'pyreneIcon-chevronUp': this.state.displayPopover, 'pyreneIcon-chevronDown': !this.state.displayPopover })}
```

Comment:
`className={clsx( styles.arrowIcon, { 'pyreneIcon-chevronUp': this.state.displayPopover, 'pyreneIcon-chevronDown': !this.state.displayPopover })}`